README.md

```diff
@@ -243,14 +243,16 @@ Compatibility between this plugin and Zephyr Enterprise is:
 
 | Plugin | Zephyr Enterprise |
 |--------|-------------------|
-| v2.5   | v7.0 or later     |
+| v2.9   | v8.1 or later    |
+| v2.8   | v7.16 to v8.0.1    |
+| v2.7   | v7.0 to v7.15.1    |
 | v2.4   | v7.0 to v7.15     |
```

Comment:
version 2.5 seems to be missing have we removed intentionally ??

Reply:
perhaps, let me check. I just synced the source and destination

<img width="1728" alt="Screenshot 2024-07-22 at 12 04 56 PM" src="https://github.com/user-attachments/assets/b24aa2b4-5c74-4e1c-bf29-00ef19b52ea3">

Reply:
@d-kafley 

This is from a previous commit
[4f46891](https://github.com/SmartBear/zephyr-enterprise-test-management-plugin/commit/4f4689188ceb1ad717b4e54fb2ce7e7191bc02cb)